src/components/Modeler.vue

```diff
@@ -515,8 +515,12 @@ export default {
     },
     removeUnsupportedElementAttributes(definition) {
       const unsupportedElements = ['documentation', 'extensionElements'];
+
       unsupportedElements.filter(name => definition.get(name))
         .forEach(name => definition.set(name, null));
```

Comment:
```suggestion
        .forEach(name => definition.set(name, undefined));
```

Reply:
Also part of this change, but GitHub currently doesn't support multi line suggestions: Remove the additional filter:
```js
definition.$descriptor.properties = definition.$descriptor.properties.filter(
        p => !unsupportedElements.includes(p.name));
```